zwave_js_server/model/driver/__init__.py

```diff
@@ -242,6 +249,38 @@ async def async_install_config_update(self) -> bool:
         )
         return cast(bool, result["success"])
 
+    async def async_firmware_update_otw(
+        self,
+        *,
+        update_data: FirmwareUpdateData | None = None,
+        update_info: FirmwareUpdateInfo | None = None,
+    ) -> DriverFirmwareUpdateResult:
+        """Send firmwareUpdateOTW command to Driver."""
+        if update_data is None and update_info is None:
+            raise ValueError(
+                "Either update_data or update_info must be provided for firmware update."
+            )
+        if update_data is not None and update_info is not None:
+            raise ValueError(
+                "Only one of update_data or update_info can be provided for firmware update."
+            )
+        params: FirmwareUpdateDataDataType | FirmwareUpdateInfoDataType
+        if update_data is not None:
+            params = update_data.to_dict()
+        elif update_info is not None:
+            params = update_info.to_dict()
```

Comment:
I think this is not correct. Z-Wave JS Server expects a property `updateInfo`:
https://github.com/zwave-js/zwave-js-server/pull/1432/files#diff-03515d137479c0dec74f922b42e9fb01e6828ea72077e82b07e0f919efae015aR103

but if I understand it correctly, this call spreads the child properties of that dict instead:
https://github.com/home-assistant-libs/zwave-js-server-python/pull/1262/files#diff-fffbb910cbecd6ea35b7baae1173e477ba1218a3def64d1aa3f045762d9ebbe1R65-R78